tests/integration/ambient/bookinfo_test.go

```diff
@@ -1,376 +0,0 @@
-	"istio.io/istio/pkg/test/framework/components/istio"
-	"istio.io/istio/pkg/test/framework/components/istioctl"
-	"istio.io/istio/pkg/test/framework/components/namespace"
-	"istio.io/istio/pkg/test/framework/resource"
-	"istio.io/istio/pkg/test/framework/resource/config/apply"
-	"istio.io/istio/pkg/test/framework/resource/config/cleanup"
-	kubetest "istio.io/istio/pkg/test/kube"
-	"istio.io/istio/pkg/test/util/assert"
-	"istio.io/istio/pkg/test/util/file"
-	"istio.io/istio/pkg/test/util/retry"
-)
-
-const (
-	bookinfoDir     = "../../../samples/bookinfo/"
-	bookinfoFile    = bookinfoDir + "platform/kube/bookinfo.yaml"
-	defaultDestRule = bookinfoDir + "networking/destination-rule-all.yaml"
-	bookinfoGateway = bookinfoDir + "networking/bookinfo-gateway.yaml"
-	routingV1       = bookinfoDir + "networking/virtual-service-all-v1.yaml"
-	headerRouting   = bookinfoDir + "networking/virtual-service-reviews-test-v2.yaml"
-	templateFile    = "manifests/charts/istio-control/istio-discovery/files/waypoint.yaml"
-)
-
-func TestBookinfo(t *testing.T) {
-	framework.
-		NewTest(t).
-		Features("traffic.ambient").
-		Run(func(t framework.TestContext) {
-			skipsForTest(t)
-			nsConfig, err := namespace.New(t, namespace.Config{
-				Prefix: "bookinfo",
-				Inject: false,
-				Labels: map[string]string{
-					constants.DataplaneMode: "ambient",
-				},
-			})
-			if err != nil {
-				t.Fatal(err)
-			}
-			setupBookinfo(t, nsConfig)
-			applyDefaultRouting(t, nsConfig)
-
-			jar, err := cookiejar.New(nil)
-			if err != nil {
-				t.Fatal(fmt.Errorf("Got error while creating cookie jar %s", err.Error()))
-			}
-			ingressClient := http.Client{}
-			ingressInst := istio.DefaultIngressOrFail(t, t)
-			addr, ingrPort := ingressInst.HTTPAddress()
-			ingressURL := fmt.Sprintf("http://%v:%v", addr, ingrPort)
-
-			t.NewSubTest("no waypoint").Run(func(t framework.TestContext) {
-				t.NewSubTest("productpage reachable").Run(func(t framework.TestContext) {
-					retry.UntilSuccessOrFail(t, func() error {
-						resp, err := ingressClient.Get(ingressURL + "/productpage")
-						if err != nil {
-							return fmt.Errorf("error fetching /productpage: %v", err)
-						}
-						defer resp.Body.Close()
-						if resp.StatusCode != http.StatusOK {
-							return fmt.Errorf("expect status code %v, got %v", http.StatusFound, resp.StatusCode)
-						}
-						bodyBytes, err := io.ReadAll(resp.Body)
-						if err != nil {
-							return fmt.Errorf("error reading /productpage response: %v", err)
-						}
-						reviewsFound := strings.Contains(string(bodyBytes), "Reviews served by:")
-						detailsFound := strings.Contains(string(bodyBytes), "Book Details")
-						if !reviewsFound || !detailsFound {
-							return fmt.Errorf("productpage could not reach other service(s), reviews reached:%v details reached:%v", reviewsFound, detailsFound)
-						}
-						return nil
-					})
-				})
-			})
-
-			t.NewSubTest("ingress receives waypoint updates").Run(func(t framework.TestContext) {
```

Comment:
Are we still covering this ingress scenario? maybe elsewhere?

Reply:
Added a test [here](https://github.com/istio/istio/pull/49353/files#diff-26abb77a151a61b70b85d966d14be3c854fa5c6a2881fa36186710e63be93f05R2157-R2161)